data-builder/build_synonyms.py

```diff
@@ -27,6 +27,13 @@ def build_synonyms():
     add(data, 'Repair Amplification', ['Repair Healing Amplification'])
     add(data, 'Well Rounded', ['Well-Rounded'])
 
+    add(data, ['Combustion', 'Radiance'], ['Power of the Flames of Purity'])
```

Comment:
i think these can be covered by `build_affix_groups.py` without having to change the parser. i was going to test it out but got distracted and now it's late. 🙂